crates/cli-flags/src/lib.rs

```diff
@@ -924,3 +961,144 @@ impl CommonOptions {
         Ok(())
     }
 }
+
+pub trait FromFile {
+    fn from_file<P: AsRef<Path>>(path: P) -> Result<Self>
+    where
+        Self: Sized;
+}
+
+impl FromFile for Config {
+    fn from_file<P: AsRef<Path>>(path: P) -> Result<Self> {
+        let file_contents = fs::read_to_string(path.as_ref())?;
```

Comment:
Mind updating the error using `.with_context(...)` to attach the name of the file here and below?

Reply:
addressed.